explorer/client/README.md

```diff
@@ -33,32 +33,46 @@ Before running any of the following scripts `yarn` must run in order to install
 
 # How to Switch Environment
 
-## Connecting to Remote Gateway server(e.g., DevNet)
+## Connecting to the DevNet Remote Gateway Server
```

Comment:
@Clay-Mysten / @Jibz1 I am cherry-picking some hotfixes on the Explorer side from the `main` branch to the `devnet` branch, which involved some minor doc changes. Just wanted to double check with you two to make sure this is okay

Reply:
@666lcz the main branch is currently serving the content